konfera/event/views.py

```diff
@@ -1,21 +1,35 @@
+import logging
+
 from datetime import timedelta
+from smtplib import SMTPException
 
+from django import VERSION
+from django.conf import settings as django_settings
```

Comment:
django_settings are there because otherwise I was unable to test behaviour under different settings